modular_skyrat/modules/synth_construction/code/designs.dm

```diff
@@ -0,0 +1,147 @@
+#define RND_CATEGORY_MECHFAB_SYNTH_PARTS "/Synthetic Parts"
+#define RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_ALL "/All Synthetics"
+#define RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_HUMANOID "/Synthetic Humanoid"
+#define RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_LIZARD "/Synthetic Lizard"
+#define RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_MAMMAL "/Synthetic Mammal"
+#define RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_IPC "/IPC"
+
+/*
+	IPC/SYNTHLIZ/SYNTH CONSTRUCTION
+	HELL YEAH LETS GET THIS BREAD
+*/
+
+/datum/design/power_cord
+	name = "Arm Power Cord"
+	desc = "Required for synthetics to recharge. Do not forget."
+	id = "arm_cord"
+	build_type = MECHFAB
+	build_path = /obj/item/organ/internal/cyberimp/arm/power_cord
+	materials = list(/datum/material/iron = 1200, /datum/material/gold = 200, /datum/material/glass = 500)
+	construction_time = 100
+	category = list(RND_CATEGORY_MECHFAB_SYNTH_PARTS + RND_SUBCATEGORY_MECHFAB_SYNTH_PARTS_ALL)
+
+/datum/design/ipc_chassis
+	name = "IPC chassis"
+	desc = "A self actualization device is recommended for converting cybernetic limbs to the correct type."
+	id = "ipc_chassis"
+	build_type = MECHFAB
+	research_icon = BODYPART_ICON_IPC
+	research_icon_state = "synth_chest"
+	build_path = /mob/living/carbon/human/species/ipc/chassis
```

Comment:
i don't think it's a good idea for the crew to be able to print torsos on demand, this trivializes sourcing a new body for decapitated people as it can now be printed without requiring genetics to be involved